src/libcmd/repl.cc

```diff
@@ -791,6 +807,27 @@ void NixRepl::addAttrsToScope(Value & attrs)
     staticEnv->sort();
     staticEnv->deduplicate();
     notice("Added %1% variables.", attrs.attrs()->size());
+
+    lastLoaded = attrs;
+
+    const int max_print = 10;
+    int counter = 0;
+    std::string loaded;
+    for (auto & i : attrs.attrs()->lexicographicOrder(state->symbols)) {
+        if (counter >= max_print)
+            break;
+
+        if (counter > 0)
+            loaded += ", ";
+
+        loaded += state->symbols[i->name];
+        counter += 1;
+    }
+
+    notice("- %1%", loaded);
```

Comment:
Not sure what the dash is for. I'd expect a dash per variable, but it's already a comma separated list
```suggestion
    notice("%1%", loaded);
```
```suggestion
    notice("  %1%", loaded);
```

Reply:
To distinguish it from fixed outputs, I'll just remove it.